PWGJE/EMCALJetTasks/AliAnalysisTaskRhoSparse.cxx

```diff
@@ -22,22 +22,26 @@ ClassImp(AliAnalysisTaskRhoSparse)
 
 //________________________________________________________________________
 AliAnalysisTaskRhoSparse::AliAnalysisTaskRhoSparse() : 
-  AliAnalysisTaskRhoBase("AliAnalysisTaskRhoSparse"),
-  fNExclLeadJets(0),
-  fExcludeOverlaps(0),
-  fRhoCMS(0),
-  fHistOccCorrvsCent(0)
+AliAnalysisTaskRhoBase("AliAnalysisTaskRhoSparse"),
+fNExclLeadJets(0),
+fExcludeOverlaps(0),
+fRhoCMS(0),
+fUseTPCArea(0),
+fExcludeAreaExcludedJets(0),
+fHistOccCorrvsCent(0)
```

Comment:
Why is this formatting applied? Code is better readable with spaces for the attribute definition.